participants/rickynilsson/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,12 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+    
+# Function is_prime that takes an integer number and returns True if the number is prime, and False if not.
+def is_prime(number):
+    if number < 2:
+        return False
+    for i in range(2, number):
```

Comment:
While this is correct, more efficient would be to limit the range to number//2.  Or, a more efficient process would be to iterate from 3 to sqrt(number) by every other integer, after checking divisibility by 2 explicitly.